yarl/_url.py

```diff
@@ -1163,6 +1162,19 @@ def update_query(self, *args: Any, **kwargs: Any) -> "URL":
             self._val._replace(query=self._get_str_query(query) or ""), encoded=True
         )
 
+    def without_query_params(self, *query_params: str) -> "URL":
```

Comment:
@bdraco @mjpieters how would you feel about forcing the args to be positional-only, API-wise?
```suggestion
    def without_query_params(self, *query_params: str, /) -> "URL":
```

Reply:
Oh thats not going to work because its unpacked
```
yarl/_url.py:1166:57: error: / must be ahead of *  [syntax]
```